components/org.wso2.carbon.identity.mgt/src/test/java/org/wso2/carbon/identity/mgt/store/test/unit/IdentityStoreTests.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
No need to update the license year if you are not introducing a new file.

Reply:
fixed with https://github.com/wso2/carbon-identity-mgt/pull/182